pages/apis/rest_api/builds.md.erb

```diff
@@ -685,11 +685,11 @@ Each <em>build</em> is provided with the following timestamps:
 <tbody>
   <tr>
     <th><code>scheduled_at</code></th>
-    <td>The time the build was created</td>
+    <td>The time the build was created. All builds from a <code>pipeline upload</code> have a <code>scheduled_at</code> copied from the job that did the uploading</td>
   </tr>
   <tr>
     <th><code>created_at</code></th>
-    <td>The time the build was created</td>
+    <td>The time the build was created. It is the same as for the <code>pipeline upload</code></td>
```

Comment:
@karensawrey 

This still isn't clear. `scheduled_at` is *set* to a time. But `pipeline upload` *happens*.
Maybe for uploaded pipelines `created at` and `scheduled at` are the same?

Reply:
@pzeballos can you clarify?

Reply:
`created_at` is nothing special, just the time the job was created. this corresponds with when the pipeline upload happened. When you `pipeline upload`, the `scheduled_at` is copied from the job that did the uploading (if that makes sense). That ensures that when we order by jobs and assign them to agents, those jobs will run first

Reply:
```suggestion
    <td>The time the build was created.  For uploaded pipelines it is when the <code>pipeline upload</code> was run.</td>
```

Reply:
THANKS @pzeballos